cmd/agent/commands/grpc.go

```diff
@@ -17,25 +18,23 @@ import (
 func BuildGrpcCmd(env runtime.Environment, config *agent.Config) *cobra.Command {
 	disruption := grpc.Disruption{}
 	var duration time.Duration
-	var port uint
-	var target uint
-	var iface string
-	var upstreamHost string
+	var port string
```

Comment:
Same as in other places, I don't see the advantage of using string when defining ports. In my opinion types communicate information about the domain.